Update contribution documents (#8382)
acemod/ACE3: docs/wiki/development/merging-pull-requests.md

order: 5
---

This page describes the process of how we merge pull requests.

> All contributors may add themselves to the AUTHORS.txt file (email address optional) if they wish.

Reviewer: ```suggestion
All contributors may add themselves to the `AUTHORS.txt` file (email address optional) if they wish.
```

The `authors` array property in `config.cpp` files will generally contain the original component author(s) and is otherwise reserved for individuals who have contributed significantly to a component (decided at the discretion of the ACE team). Do not add yourself without consent.

> The `author` string property in `config.cpp` files will always contain the common ACE team string (`ECSTRING(common,ACETeam)`) to reflect component maintanence and keep things consistent.

Author: ```suggestion
The `author` string property in `config.cpp` files will always contain the common ACE team string (`ECSTRING(common,ACETeam)`) to reflect component maintenance and keep things consistent.
```

#### Merge Criteria

> All pull requests must receive approval from a maintainer and must pass all checks (GitHub Actions) before they can be merged.

Reviewer: ```suggestion
All pull requests must receive approval from a maintainer and must pass all required continuous integration checks before they can be merged.
```

> All pull requests must receive approval from a maintainer and must pass all required continuous integration checks before they can be merged.

Reviewer 2: Should this be `members` instead of `maintainers` now?

...then the PR can be merged right away by one of the maintainers.

> Non-trivial pull requests should ideally be thouroughly reviewed by multiple maintainers or at least one maintainer highly familiar with any code modified.

Reviewer 2: Same
